products/sle/main.pm

```diff
@@ -914,6 +914,7 @@ sub load_fips_tests_misc() {
     loadtest "console/journald_fss.pm";
     loadtest "fips/curl_fips_rc4_seed.pm";
     loadtest "console/consoletest_finish.pm";
+    loadtest "console/git_fips.pm";
```

Comment:
again, I don't see this as fips specific. I understand you wrote it with fips in mind and it should run in the fips_tests. But the test itself just tests git and nothing about fips.

So please just name it git and load the test in opensuse/main.pm and sle/main.pm as console test.